packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -3699,6 +3699,25 @@ describe('network stubbing', { retries: 15 }, function () {
       .wait('@a').its('response.body').should('eq', 'bar')
     })
 
+    // @see https://github.com/cypress-io/cypress/issues/25448
+    it('gets all aliased Interceptions by alias.all when assigning an alias using req.alias', function () {
```

Comment:
Can we move this into the ` context('cy.get(alias)', function () {` context next to the existing test `it('gets all aliased Interceptions by alias.all', function () {`?

Reply:
Yup! Done in 9682cbbdbc9bb6a40856a7861196f01d7e45b5f0